src/SDKs/ManagedServiceIdentity/ManagedServiceIdentity.Tests/ManagedServiceIdentity.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>ManagedServiceIdentity.Tests</PackageId>
+    <Description>ManagedServiceIdentity.Tests Class Library</Description>
+    <AssemblyName>ManagedServiceIdentity.Tests</AssemblyName>
+    <VersionPrefix>1.0.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Management.ManagedServiceIdentity\Microsoft.Azure.Management.ManagedServiceIdentity.csproj" />
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```

Comment:
@varunkch use wildcard to add all json files in your project. see an example
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L20

Reply:
Done.